examples/simple/test/SimpleState.t.sol

```diff
@@ -38,10 +38,28 @@ contract SimpleStateTest is SymTest, Test {
 
         // note: a total of 253 feasible paths are generated, of which only 10 unique states exist
         for (uint i = 0; i < 10; i++) {
-            (success,) = address(target).call(svm.createCalldata("SimpleState"));
+            (success,) = address(target).call(svm.createCalldata("SimpleState")); // excluding view functions
             vm.assume(success);
         }
 
         assertFalse(target.buggy());
     }
+
+    function check_buggy_with_snapshot() public {
+        bool success;
+
+        // take the initial storage snapshot
+        uint id = svm.snapshotStorage(address(target));
+
+        // note: a total of 253 feasible paths are generated, of which only 10 unique states exist
+        for (uint i = 0; i < 10; i++) {
+            (success,) = address(target).call(svm.createCalldata("SimpleState", true)); // including view functions
+            vm.assume(success);
+            uint nid = svm.snapshotStorage(address(target));
+            vm.assume(nid > id); // ignore if no state changes
```

Comment:
is there a reason to use `>` rather than `!=`?

Reply:
ok I see it's an auto-incrementing id, but IMO we should probably model that these are just opaque values that are meant to be compared for equality, not ordered